### DISCUSSION TITLE
Download

### DISCUSSION THREAD

**Discussion by 7Katta** [chromiecraft/chromiecraft#5291] [General]
Is it possible to download the client without using torrent?
Or is there another p-server whose client I can download and then enter Chromie's realmlist info?

Thank you for your assistance

**Comment by 55Honey** [comment]
Hi @7Katta !
There's a direct download at https://www.chromiecraft.com/en/downloads/ to the right of the download section.

Consider joining our discord for better user interaction. See you around!

**Reply by 7Katta** [reply]
I appreciate the quick response! I had noticed that direct download link, but when I started it (after copying the link to the address bar) it appeared to be using torrent to download (seeding, peers, etc.).  I don't use a VPN and don't wish to install one, so I am concerned about security. Just wanting to be sure I practice safe, er ... downloading. 

If I can get the client downloaded I will definitely join the Discord community!

**Reply by 55Honey** [reply]
Right click this 
![image](https://user-images.githubusercontent.com/71938210/228964721-fbdad951-c3de-412d-ac73-a088b13b4dfa.png)
and select "Save As"?

**Reply by 7Katta** [reply]
I've got it downloaded now, thank you! 
And I forgot I had already joined the Discord community, so will be interacting there.
Thank you for your help!
